jlibra-core/src/main/java/dev/jlibra/move/Move.java

```diff
@@ -1,18 +1,38 @@
 package dev.jlibra.move;
 
-import com.google.protobuf.ByteString;
-
-import org.bouncycastle.util.encoders.Hex;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
 
 public class Move {
 
-    private static final String peerToPeerTransferBinary =
-            "4c49425241564d0a010007014a00000004000000034e000000060000000c5" +
-            "4000000060000000d5a000000060000000560000000290000000489000000" +
-            "2000000007a90000000e00000000000001000200010300020002040200030" +
-            "003020402063c53454c463e0c4c696272614163636f756e74046d61696e0f" +
-            "7061795f66726f6d5f73656e6465720000000000000000000000000000000" +
-            "0000000000000000000000000000000000001020104000c000c0111010002";
+    public static byte[] peerToPeerTransferAsBytes() {
+        try {
+            InputStream jsonCode = Move.class.getResourceAsStream("/move/peer_to_peer_transfer.bin.json");
+            String json = readFullyAsString(jsonCode, "UTF-8");
+            String[] bytesAsString = json.substring(json.indexOf("[")+1, json.indexOf("]")).split(",");
+            byte[] bytes = new byte[bytesAsString.length];
+            int idx = 0;
+            for (String byteAsString : bytesAsString) {
+                bytes[idx++] = (byte)(Integer.valueOf(byteAsString).byteValue() & 0xFF);
+            }
+            return bytes;
+        } catch (Exception ex) {
```

Comment:
I think there is only `IOException` to catch here, no need for generic Exception clause

Reply:
Ok.